Localization.EntityFramework/EFStringLocalizer.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using EFLocalizationSample.Model;
+using Microsoft.Data.Entity;
+using Microsoft.Extensions.Localization;
+
+namespace EFLocalizationSample
+{
+    public class EFStringLocalizer : IStringLocalizer
+    {
+        private readonly LocalizationDBContext _db;
+
+        public EFStringLocalizer(LocalizationDBContext db)
+        {
+            _db = db;
+        }
+
+        public LocalizedString this[string name]
+        {
+            get
+            {
+                var value = GetString(name);
+                return new LocalizedString(name, value ?? name, resourceNotFound: value == null);
+            }
+        }
+
+        public LocalizedString this[string name, params object[] arguments]
+        {
+            get
+            {
+                var format = GetString(name);
+                var value = string.Format(format ?? name, arguments);
+                return new LocalizedString(name, value, resourceNotFound: format == null);
+            }
+        }
+
+        public IStringLocalizer WithCulture(CultureInfo culture)
+        {
+            CultureInfo.DefaultThreadCurrentCulture = culture;
+            return new EFStringLocalizer(_db);
+        }
+
+        public IEnumerable<LocalizedString> GetAllStrings(bool includeAncestorCultures)
+        {
+            return _db.Resources.Include(r => r.Culture).Where(r => r.Culture.Name == CultureInfo.CurrentCulture.Name)
```

Comment:
Recommend starting a new line on every new LINQ statement:

```c#
return _db.Resources
    .Include(r => r.Culture)
    .Where(r => r.Culture.Name == CultureInfo.CurrentCulture.Name)
    .Select(r => new LocalizedString(r.Key, r.Value, true)); // and use a named param for the bool
```

Please do this throughout the code in this PR.